ec2/spark_ec2.py

```diff
@@ -178,6 +178,9 @@ def parse_args():
         "-i", "--identity-file",
         help="SSH private key file to use for logging into instances")
     parser.add_option(
+        "-p", "--profile", default="default",
+        help=" If you have multiple profiles, you can configure additional, named profiles by using this option (default: %default)")
```

Comment:
Can you expand this to explain what type of profile this is talking about? Like, maybe clarify whether it's an IAM or boto profile or whatever? As it stands now, I worry that this description isn't clear to people unfamiliar with this feature.